modules/client-configuration/pages/registration-pubcloud.adoc

```diff
@@ -0,0 +1,60 @@
+[[registering.clients.pubcloud]]
+= Register Public Cloud Clients
+
+If you have your clients running in a public cloud instance, there are several methods for registering them to your {productname} Server, in addition to the regular bootstrapping methods.
+For other registration methods, see xref:client-configuration:registration-overview.adoc[].
+
+By default, logging for both the {productname} Server and Salt clients is disabled.
+You can enable logging on Salt clients to provide some additional information during registration.
+Open or create the [path]``/etc/salt/minion.d/log_level.conf`` configuration file on the client, and add or edit this line:
+
+----
+log_level: info
+----
+
+
+
+== Register Salt Clients in the Instance
+
+This method makes the Salt client part of the public cloud image.
+
+
+
+.Procedure: Registering Salt Clients in the Instance 
+. Before you begin, ensure you have an activation key prepared on the {productname} Server.
+. Set the fully-qualified domain name (FQDN) of the client.
+. On the client, at the command prompt, open or create the [path]``/etc/salt/minion.d/susemanager.conf`` file, and add or edit it to include:
++
+----
+master: SERVER.EXAMPLE.COM
+server_id_use_src: adler32
+enable_legacy_startup_events: False
+enable_fqdns_grains: False
+grains:
+    susemanager:
+        activation_key: <ACTIVATION_KEY_NAME>
+----
+. Restart the [systemitem]``salt-minion`` service:
++
+----
+systemctl restart salt-minion
+----
+. On the {productname} Server, accept the new client key; replace [systemitem]``<client>`` with the name of your client:
++
+----
+salt-key -a '<client>'
+----
+
+
+
+== Register Clients with SSH
+
+You can use the ``mgr-ssh-push-init`` script to register both traditional and Salt clients with SSH, or SSH tunnelling.
+For a copy of the script, see xref:client-configuration:registration-pubcloud-script.adoc[].
+
+
+
+== Register Clients Using the API
+
+You can also use the API to register clients.
+For the most recent API documentation, see https://documentation.suse.com/suma.
```

Comment:
Maybe we can be a bit more specific. 

The API calls are 

- bootstrap
- bootstrapWithPrivateSshKey

Reply:
Added